### ISSUE TITLE
Error thrown on c:strike_select picklist when searchable="true"

### ISSUE THREAD

**Issue by patrick-fischer** [appiphony/Strike-Components#38]
The strike_select cmp throws the following error when inputting values (that return any result) in the searchable field:
`Access Check Failed! AttributeSet.get(): attribute 'hidden' of component 'markup://c:strike_optionGroup {42:0}' is not visible to 'markup://c:strike_select {3:0}'.
Failing descriptor: {markup://c:strike_select}`

I use the default c:strike_select (master) as the sole component with the addition of a searchable="true" attribute:
`<c:strike_select value=""
                 label="Select Label"
                 placeholder="Select an Option"
                 errorMessage="Invalid input"
                 searchable="true">
    <c:strike_option label="Option 1" value="option-1"/>
    <c:strike_option label="Option 2" value="option-2"/>
</c:strike_select>`

Reproduce by adding the above component to an empty lightning app and by entering 'o' into the searchable input field.

**Comment by justinAppiphony** [comment]
Hi @patrick-fischer, we have not been able to duplicate this error. Have you gotten it to work? We have since updated to version 40.

Thank you,
Justin

**Comment by justinAppiphony** [comment]
@patrick-fischer Can you be a little more specific with what you mean by in an empty app?
Thank you,
Justin

**Comment by patrick-fischer** [comment]
TLDR: I am referring to the aura:application tag, @justinAppiphony.


We develop native Lightning applications. Once switched on Lightning in an org, we use the Dev-Console to create File->Lightning Application->setting a Name. Then we extend the app using force:slds and we get something like:
`<aura:application extends="force:slds" >
</aura:application>`

I add any <c:strike_* > inside the aura:application tag and then hit 'Update Preview'. For issue (2), it would look like the following:
`<aura:application extends="force:slds" ><c:strike_select value="" label="Select Label" placeholder="Select an Option" errorMessage="Invalid input" searchable="true"> <c:strike_option label="Option 1" value="option-1"/> <c:strike_option label="Option 2" value="option-2"/> </c:strike_select></aura:application>`

Therefore, with 'empty app' I was meaning that this app has <c:strike_*> as the sole component inside aura:application .

**Comment by justinAppiphony** [comment]
@patrick-fischer I followed your instructions and it is working here on our end. I'll give you a quick step-by-step snapshot just to be sure.

Step 1) New Lightning app
File > New > Lightning Application

Step 2)
![image](https://user-images.githubusercontent.com/882708/28897552-e2f8a626-77a6-11e7-973f-f3f948d36792.png)

**Comment by patrick-fischer** [comment]
@justinAppiphony So if you enter a few characters into the search box (like 'Opti') and if you subsequently use backspace to remove those char's then you can't reproduce the issue?

**Comment by justinAppiphony** [comment]
Hi @patrick-fischer we do not see an error on our end.

**Comment by justinAppiphony** [comment]
@patrick-fischer We have looked at it again and have not been able to reproduce that error. For now we will be closing this issue.

Thank you,
Justin
